backend/interfaces/ideas/router.py

```diff
@@ -62,6 +63,10 @@ async def get_idea(id: int, db: Session = Depends(get_db)):
 
 @router.post('', response_model=IdeaResponse)
 async def create_idea(idea_request: CreateIdea, db: Session = Depends(get_db)):
+    for word in BANNEDWORDS:
+        if idea_request.title == word or idea_request.description == word or idea_request.tags == word:
```

Comment:
i would rather check it like this:

```
for word in BANNEDWORDS:
    lower_word = word.lower()
    if lower_word in idea_request.title.lower() or lower_word in idea_request.description.lower() or lower_word in idea_request.tags.lower():

```

Reply:
oh yeah that makes more sense,
let me fix that quickly!

Reply:
fixed!